experimental/displaynames/src/provider.rs

```diff
@@ -19,7 +20,7 @@ use zerovec::ZeroMap;
 // become inaccessible).
 type UnvalidatedRegion = TinyAsciiStr<3>;
 
-#[icu_provider::data_struct(TerritoryDisplayNamesV1Marker = "displaynames/territories@1")]
+#[icu_provider::data_struct(TerritoryDisplayNamesV1Marker = "territory_displaynames/territories@1")]
```

Comment:
Why change the names? Every byte counts and there's no point in including "territory" (and language below) twice.

Reply:
Makes sense. Done!